tests/ModelContextProtocol.Tests/Transport/StdioServerTransportTests.cs

```diff
@@ -32,11 +31,9 @@ public StdioServerTransportTests(ITestOutputHelper testOutputHelper)
         };
     }
 
-    [Fact]
+    [Fact(Skip="https://github.com/modelcontextprotocol/csharp-sdk/issues/143")]
```

Comment:
If it only occurs on Ubuntu, might make sense to skip conditionally. Xunit.v3 has `Assert.Skip` and `Assert.SkipWhen` that makes this straightforward.

Reply:
Oh, I just saw this literally undoes a `SkipWhen`. I take it this wasn't platform specific in the end?

Reply:
Sorry. I hid this in the commit message:

```
- Canceling read from stdin in StdioServerTransport.DisposeAsync is also flaky on Windows
- https://github.com/modelcontextprotocol/csharp-sdk/actions/runs/14161333333/job/39667212874
```